### PR TITLE
airflow 2 changes

### PR DESCRIPTION
**Pull request by omerharoonedu**
Adding changes to make brokkr compatible with airflow 2.

### CODE REVIEW

File: plugins/airflow/airflow.mk [hunk 1]
```diff
@@ -16,96 +17,96 @@ AIRFLOW_REQUIREMENTS_EXTRA_TXT ?= requirements.extra.txt
 
 .PHONY: airflow.start
 airflow.start: $(AIRFLOW_BUILD_SENTINEL) $(AIRFLOW_VARIABLES_SENTINEL) ## Start Airflow server
-	docker-compose -f $(AIRFLOW_DOCKER_COMPOSE_FILE) up -d db
-	docker-compose -f $(AIRFLOW_DOCKER_COMPOSE_FILE) up -d webserver
-	docker-compose -f $(AIRFLOW_DOCKER_COMPOSE_FILE) up -d scheduler
-	echo "Now running at http://localhost:$(shell grep "AIRFLOW_WEBSERVER_PORT" .env | cut -d "=" -f2)/"
+    docker-compose -f $(AIRFLOW_DOCKER_COMPOSE_FILE) up -d db
```
Comment (judoole) [inline plugins/airflow/airflow.mk]: A lot of indentation changes. Is this some setting on your IDE @omerharoonedu, or is it the recommended setting?

Comment (judoole) [reply plugins/airflow/airflow.mk]: It is a bit hard to see what has actually changed, when there are all of these indentations 😄

**Review by judoole — APPROVED** [review]

File: plugins/airflow/airflow.mk (hunk 1 above)
Comment (omerharoonedu) [reply plugins/airflow/airflow.mk]: I have no idea why this happened, it didn't actually show on the diff in VS code. XD

Comment (omerharoonedu) [reply plugins/airflow/airflow.mk]: Fixed it.

Comment (torbjornvatn) [reply plugins/airflow/airflow.mk]: Screenshot 2022-03-17 at 09 12 55

It's actually possible to hide the whitespace in the diff here. 
But this looks like a classical Spaces vs. Tabs for indentation issue. What's the settings for `Editor: Tab Size`, `Editor: Insert Spaces` and `Editor: Detect Indentation` @omerharoonedu ?

Comment (torbjornvatn) [reply plugins/airflow/airflow.mk]: And from what I can see, VS Code is choosing Tabs as the default indentation for Makefile

Screenshot 2022-03-17 at 09 57 44
s

**Comment by torbjornvatn** [comment]
@omerharoonedu and @judoole I've been pondering this a bit, and maybe it's not `AIRFLOW_MAJOR_VERSION -eq "2"` that is the correct thing to check for in the if? Because the reason we need this if is that the official airflow docker image has a different `ENTRYPOINT` (the airflow binary) than or old homegrown one.

So maybe the check here should just be `OFFICIAL_AIRFLOW_IMAGE -eq "true"` or something

**Comment by torbjornvatn** [comment]
Talked to @omerharoonedu about this now, and it is in fact reliant on Airflow2 because of the differences in syntax.
So never mind my comment 🤷🏼 

Should we maybe release a new version with this in it then?

**Comment by torbjornvatn** [comment]
Or Omer is actually updating the default Dockerfiles here in the repo now to have support for both Airflow 1 and 2

**Comment by judoole** [comment]
Yeah, maybe we should just make it only Airflow 2. Soon, none of our internal projects will be using Airflow 1.

**Review by torbjornvatn — APPROVED** [review]
Should we just try making a new release @judoole ?

**Pull request merged**